uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-calendar-day.component.spec.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'leave-absences/shared/components/leave-calendar-day.component'
+], function () {
+  'use strict';
+
+  describe('leaveCalendarDay', function () {
+    var $componentController, $log;
+
+    beforeEach(module('leave-absences.templates', 'leave-absences.components'));
+    beforeEach(inject(function (_$componentController_, _$log_) {
```

Comment:
Should there be a line break? I realise these are 2 function calls, but we separate jasmine calls with an empty line usually?